samples/net/sockets/echo_client/overlay-s2868a1.conf

```diff
@@ -0,0 +1,14 @@
+CONFIG_GPIO=y
```

Comment:
Once again, please follow up on https://github.com/zephyrproject-rtos/zephyr/pull/14057/commits
shield overlays would now sit under boards/shields, so they are not tight to a sample